projectatomic/ai-studio
# feat: async creation of playground

### What does this PR do?

Creates the playground async, and use tasks to display to the user the advancement

Out of scope for this PR:
- A refactoring is possible as the application, inference server and playground creations use the same technique.

### Screenshot / video of UI


### What issues does this PR fix or reference?

Part of #525 
Fixes #615 

### How to test this PR?

Delete inference servers and related image, and create a new playground.

The creation page should display a task 'Creating...'

When the playground is created, a button should be available to go to the Playground page
